docs/content/getting-started/docker.md

```diff
@@ -48,3 +48,17 @@ For example, for running cardano-node + ogmios on the testnet, listening to tcp/
 ```console
 $ NETWORK=testnet OGMIOS_PORT=1338 docker-compose --project-name cardano-ogmios-testnet up
 ```
+
+## As a single image
+
+If you're not fond of docker-compose and the container orchestration it requires, you may also build (or pull) an image to run both a `cardano-node` and `ogmios` side-by-side, within the same container. Note that this is the default build target when building ogmios with Docker, but we can be explicit when building the image using `--target cardano-node-ogmios`. For example:
+
+```console
+$ docker build --target cardano-node-ogmios --tag ktorz/cardano-node-ogmios:latest . 
```

Comment:
```suggestion
$ docker build --tag ktorz/cardano-node-ogmios:latest . 
```

Reply:
Think it's best to use no target here, then add a note below

Reply:
Ideally though, using a remote build context means a consumer can run a build command without cloning the repo first. 

This is [valid](https://docs.docker.com/engine/reference/commandline/build/#git-repositories):
```
$ DOCKER_BUILDKIT=1 docker build --cache-from ktorz/ogmios:latest --tag ktorz/cardano-node-ogmios:latest https://github.com/KtorZ/cardano-ogmios.git#:server
```

... however [Buildkit doesn't support sub-directories](https://github.com/moby/buildkit/issues/1684) :-1: 

Considering the Docker image now represents more than just the Ogmios Server, it does hint at hoisting it to the top level. This is similar to how Cardano Rosetta was implemented: https://github.com/input-output-hk/cardano-rosetta. Is that an option?

Reply:
Awe :( ... Well, it's also possible to move the `Dockerfile` one level up indeed, though I like it well scoped within `server`. The compose file is already at the root level however, and building it without cloning the repository sounds like an argument convincing enough to move it there :+1: